lib/jxl/enc_bit_writer.h

```diff
@@ -50,29 +50,28 @@ struct BitWriter {
 
   Span<const uint8_t> GetSpan() const {
     // Callers must ensure byte alignment to avoid uninitialized bits.
-    JXL_ASSERT(bits_written_ % kBitsPerByte == 0);
-    return Bytes(storage_.data(), bits_written_ / kBitsPerByte);
+    JXL_DASSERT(bits_written_ % kBitsPerByte == 0);
+    return Bytes(storage_.data(), DivCeil(bits_written_, kBitsPerByte));
```

Comment:
I've checked all call sites - they all do care about zero-padding; given it was JXL_ASSERT before, DivCeil sounds safer - we do not drop any bits we have written (in case zero-padding is forgotten)

Reply:
But we give the caller possibly-uninitialized bits :-)

Reply:
Well, IIRC, we initialize "bleeding" byte with 0,... but perhaps not in all scenarios. Have to check.